### PR TITLE
[Beta]:docs(cn): translation api reactdom done

### PR DESCRIPTION
**Pull request by liu-jin-yi**

### CODE REVIEW

File: beta/src/pages/reference/reactdom.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 title: ReactDOM API
 layout: API
+translators:
+  - liu-jin-yi
 ---
 
 <Intro>
 
-The ReactDOM package lets you render React components on a webpage.
+ReactDOM 包能够让你在页面上渲染 React 组件。
```
Comment (evan-lc) [inline beta/src/pages/reference/reactdom.md:10]:
```suggestion
ReactDOM package 能够让你在页面上渲染 React 组件。
```

File: beta/src/pages/reference/reactdom.md [hunk 2]
```diff
@@ -22,27 +24,27 @@ npm install react-dom
...
-ReactDOM supports all popular browsers, including Internet Explorer 9 and above. [Some polyfills are required](http://todo%20link%20to%20js%20environment%20requirements/) for older browsers such as IE 9 and IE 10.
+ReactDOM 支持所有流行的浏览器，包括 Internet Explorer 9 及以上。对于 IE 9 和 IE 10 等旧浏览器 [需要一些 polyfills](http://todo%20link%20to%20js%20environment%20requirements/) 。
 
-## Exports
+## 导出
 
 <YouWillLearnCard title="render" path="/reference/render">
 
-Renders a piece of JSX ("React element") into a browser DOM container node.
+可以将一段 JSX （"React 元素"）渲染到浏览器 DOM 容器中。
```
Comment (evan-lc) [inline beta/src/pages/reference/reactdom.md:46]:
```suggestion
可以将一段 JSX （“React 元素”）渲染到浏览器 DOM 容器节点中。
```

File: beta/src/pages/reference/reactdom.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
 <Intro>
 
-The ReactDOM package lets you render React components on a webpage.
+ReactDOM 包能够让你在页面上渲染 React 组件。
 
 </Intro>
 
-Typically, you will use ReactDOM at the top level of your app to display your components. You will either use it directly or a [framework](/learn/start-a-new-react-project#building-with-react-and-a-framework) may do it for you. Most of your components should *not* need to import this module.
+通常情况下，你会在你的应用程序的顶层使用 ReactDOM 来渲染你的组件。你要么直接使用它，要么交由 [框架](/learn/start-a-new-react-project#building-with-react-and-a-framework) 为你渲染。你的大部分组件应该 *不* 需要导入这个模块。
```
Comment (evan-lc) [inline beta/src/pages/reference/reactdom.md:14]:
```suggestion
通常情况下，你会在你的应用程序的顶层使用 ReactDOM 来展示你的组件。你要么直接使用它，要么交由 [框架](/learn/start-a-new-react-project#building-with-react-and-a-framework) 为你使用。你的大部分组件应该 *不* 需要导入这个模块。
```

File: beta/src/pages/reference/reactdom.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
 
 <Intro>
 
-The ReactDOM package lets you render React components on a webpage.
+ReactDOM package 能够让你在页面上渲染 React 组件。
 
 </Intro>
 
-Typically, you will use ReactDOM at the top level of your app to display your components. You will either use it directly or a [framework](/learn/start-a-new-react-project#building-with-react-and-a-framework) may do it for you. Most of your components should *not* need to import this module.
+通常情况下，你会在你的应用程序的顶层使用 ReactDOM 来展示你的组件。你要么直接使用它，要么交由 [框架](/learn/start-a-new-react-project#building-with-react-and-a-framework) 为你使用。你的大部分组件应该 *不* 需要导入这个模块。
```
Comment (awxiaoxian2020) [inline beta/src/pages/reference/reactdom.md:14]:
```suggestion
通常情况下，你会在你的应用程序的顶层使用 ReactDOM 来展示你的组件。你要么直接使用它，要么让 [框架](/learn/start-a-new-react-project#building-with-react-and-a-framework) 为你所用。你的大部分组件应该 *不* 需要导入这个模块。
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

**Review by awxiaoxian2020 — APPROVED** [review]

**Comment by awxiaoxian2020** [comment]
I think this PR should be closed. Please let me know if I'm wrong. :)

**Pull request closed without merging**